python/dataprep.py

```diff
@@ -25,42 +25,83 @@
 METADATA_LIST = os.getenv('METADATA_LIST').split()
 
 
-# **** RESHAPE data long -> wide by taxon *******
-# ***********************************************
+def load_labelled(SINCE_THRESHOLD, level='level2'):
+    if level=='agnostic' or level=='level1':
+        dataframe = pd.read_csv(
+        os.path.join(DATADIR, 'labelled.csv.gz'),
+        dtype=object,
+        compression='gzip'
+         )
+    else:
+        dataframe = pd.read_csv(
+        os.path.join(DATADIR, 'labelled_level2.csv.gz'),
+        dtype=object,
+        compression='gzip'
+    )
```

Comment:
The indentation is a bit inconsistent here. Also, given that only the file parameter is different with these two `pd.read_csv` calls, you could perhaps just put the filename in a `if` statement, and move the `pd.read_csv` statement out.